aten/src/ATen/native/BinaryOps.h

```diff
@@ -7,6 +7,23 @@ namespace at { struct TensorIterator; }
 
 namespace at { namespace native {
 
+inline void alpha_check(const ScalarType dtype, Scalar alpha) {
```

Comment:
It's nice to know what the operation is in the error message. But since this is not making the existing code any worse than it is, this is not a blocker.